code/controllers/subsystem/processing/newtonian_movement.dm

```diff
@@ -0,0 +1,5 @@
+/// The subsystem used to tick [/datum/component/jetpack] instances.
```

Comment:
this documentation is bad, this could change or this subsystem could be expanded on, you should describe its role conceptually rather then mechanically